src/Microsoft.AspNet.Mvc.TagHelpers/LabelTagHelper.cs

```diff
@@ -15,11 +15,15 @@ public class LabelTagHelper : TagHelper
     {
         private const string ForAttributeName = "asp-for";
 
-        [Activate]
+        public LabelTagHelper(IHtmlGenerator generator)
+        {
+            Generator = generator;
+        }
+
         [HtmlAttributeNotBound]
+        [ViewContext]
         public ViewContext ViewContext { get; set; }
 
-        [Activate]
         [HtmlAttributeNotBound]
         public IHtmlGenerator Generator { get; set; }
```

Comment:
Yeah I sent the PR after realizing that I hadn't changed it. You could say I'm conserving energy by getting the feedback first :+1: